.github/workflows/integration-test-helm-suite.yaml

```
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes-versions : ['v1.15.12', 'v1.21.0']
+        kubernetes-versions : ['v1.16.15', 'v1.22.0']
```

```suggestion
        kubernetes-versions : ['v1.15.12', 'v1.22.0']
```
same multiple places where `v1.15.12` has been changed